### DISCUSSION TITLE
SyntaxError: Invalid or unexpected token in AdonisJS V6 serve command

### DISCUSSION THREAD

**Discussion by abidr** [adonisjs/core#4760] [Help] [answered]
Hello there, I'm getting the following error when running the node ace serve command. Could anyone explain what the issue is?
It was working on my other computer before. Tried deleting lock files and node_modules but no luck.

![image](https://github.com/user-attachments/assets/47663e0d-59dd-4c9e-9508-559ecd01145c)

**Comment by RomainLanz** [comment, marked as answer]
Hey @abidr! 👋🏻 

Please downgrade `@swc/core` to `1.7.26`, it seems they pushed a broken release.

**Reply by abidr** [reply]
Thank you. It worked.
